rotkehlchen/tests/utils/blockchain.py

```diff
@@ -652,6 +653,11 @@ def mock_base_has_activity(account):
             return EtherscanHasChainActivity.TRANSACTIONS
         return EtherscanHasChainActivity.NONE
 
+    def mock_gnosis_has_activity(account):
+        if account in gnosis_addresses:
+            return EtherscanHasChainActivity.TRANSACTIONS
+        return EtherscanHasChainActivity.NONE
+
```

Comment:
Maybe can you abstract this to a function that takes as argument account and chain and depending on the chain just check one list or the other? then you can use the same function in all the contexts